platforms/__common.sh

```diff
@@ -139,6 +139,7 @@ buildPlatform() {
 	local funcToCall="$2"
 	while IFS= read -r VERSION_INFO || [[ -n $VERSION_INFO ]]
 	do
+		VERSION_INFO="$(echo -e "${VERSION_INFO}" | sed -e 's/^[[:space:]]*//')"
```

Comment:
nit: would you mind adding a comment above this line to explain what's happening to `VERSION_INFO`?